netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -104,12 +104,19 @@
   private static final Status EXHAUSTED_STREAMS_STATUS =
           Status.UNAVAILABLE.withDescription("Stream IDs have been exhausted");
 
+  private static final int BDP_MEASUREMENT_PING = 1234;
+  private static final int MAX_WINDOW_SIZE = 8 * 1024 * 1024;
```

Comment:
@mfcripps I think those two static fields should also be moved to `AbstractNettyHandler` or `Utils`. Also I think it makes sense to declare `BDP_MEASUREMENT_PING` a ByteBuf and reuse that for every ping.